target/scripts/startup/setup-stack.sh

```diff
@@ -1183,6 +1183,44 @@ EOF
   supervisorctl update
 }
 
+function _setup_getmail
+{
+  _log 'trace' 'Preparing Getmail configuration'
+
+  local CONFIGURATION GETMAILRC ID CONFIGS
+
+  CONFIGURATION='/tmp/docker-mailserver/getmail-*.cf'
+  GETMAILRC='/etc/getmailrc.d'
+  CONFIGS=false
+
+  if [[ ! -d ${GETMAILRC} ]]
+  then
+    mkdir "${GETMAILRC}"
+  fi
+
+  # Generate getmailrc configs, starting with the `/etc/getmailrc_general` base config,
+  # Add a unique `message_log` config, then append users own config to the end.
+  for FILE in ${CONFIGURATION}; do
```

Comment:
```suggestion
  for FILE in /tmp/docker-mailserver/getmail-*.cf
```

This is actually idiomatic in Bash. Iterating over a variable that just happens to expand into a string with whitespaces is not, especially because it will not work when you start properly quoting it :)

Reply:
Makes sense to me.